variables.tf

```diff
@@ -56,20 +30,37 @@ variable "docker_machine_spot_price_bid" {
   default     = "0.1"
 }
 
+variable "docker_machine_user" {
```

Comment:
This variable has no any references in code

Reply:
fixed